qparam_test.go

```diff
@@ -8,8 +8,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/stoewer/go-qparam"
-	"github.com/stoewer/go-strcase"
+	qparam "github.com/stoewer/go-qparam"
+	strcase "github.com/stoewer/go-strcase"
```

Comment:
The package name is already `qparam`, or `strcase` respectively, putting an alias in front of the import is not necessary and redundant. Therefore I prefer the version without alias.